shared/common-adapters/text.desktop.js

```diff
@@ -53,14 +53,20 @@ export default class Text extends Component {
       'TerminalUsername': {...styles.textTerminal, color: globalColorsDZ2.orange},
       'TerminalPublic': {...styles.textTerminal, color: globalColorsDZ2.yellowGreen2},
       'TerminalPrivate': {...styles.textTerminal, color: globalColorsDZ2.darkBlue2},
-      'TerminalEmpty': {...styles.textTerminal, ...styles.textTerminalEmpty}
+      'TerminalEmpty': {...styles.textTerminal, ...styles.textTerminalEmpty},
+      'InputHeader': styles.textInputHeader
     }[this.props.type]
 
     const opacity = this.props.darkMode ? 1 : 0.75
 
     const color = this.props.darkMode ? globalColorsDZ2.white : globalColorsDZ2.black
 
-    const inlineStyle = this.props.inline ? this._inlineStyle(this.props.type) : {}
+    let inline = true
```

Comment:
This is the only case we have so far where if you don't specify a prop it defaults to true, I think that's confusing.

Why not default to false?